package.json

```
@@ -72,6 +72,7 @@
     "@types/pluralize": "0.0.33",
     "@types/react": "18.2.37",
     "@types/react-dom": "18.2.15",
+    "@types/react-plotly.js": "^2.6.3",
```

Can you do npm install again?
```suggestion
    "@types/react-plotly.js": "2.6.3",
```